Promise.js

```diff
@@ -126,7 +108,7 @@
 	};
 
 	Promise.all = function () {
-		var args = Array.prototype.slice.call(arguments.length === 1 && isArray(arguments[0]) ? arguments[0] : arguments);
+		var args = Array.prototype.slice.call(arguments.length === 1 && Array.isArray(arguments[0]) ? arguments[0] : arguments);
```

Comment:
@domenic Thanks for checking this out! I think issues were disabled automatically since this is a fork. I think the [upstream](https://github.com/taylorhakes/promise-polyfill) would be interested in an issue, though.

Is there an ES6 equivalent to the A+ specs that we can throw at this to test compliance?